src/toil/job.py

```diff
@@ -667,18 +667,20 @@ def _cleanLocalTempDir(self, cacheSize):
             """
             #Remove files so that the total cached files are smaller than a cacheSize
             
-            #List of pairs of (fileSize, fileStoreID) for cached files
-            cachedFileSizes = map(lambda x : (os.stat(self.jobStoreFileIDToCacheLocation[x]).st_size, x), 
-                                  self.jobStoreFileIDToCacheLocation.keys())
+            #List of pairs of (fileCreateTime, fileStoreID) for cached files
+            cachedFileCreateTimes = map(lambda x : (os.stat(self.jobStoreFileIDToCacheLocation[x]).st_ctime, x),
+                                        self.jobStoreFileIDToCacheLocation.keys())
             #Total number of bytes stored in cached files
-            totalCachedFileSizes = sum(map(lambda x : x[0], cachedFileSizes))
-            #Remove smallest files first - this is not obviously best, could do it a different
-            #way
-            cachedFileSizes.sort()
-            cachedFileSizes.reverse()
+            totalCachedFileSizes = sum([os.stat(self.jobStoreFileIDToCacheLocation[x]).st_size for x in \
```

Comment:
Backslash is not needed to continue a line within balanced constructs like brackets and parens.